docs/node-js-notes.md

```diff
@@ -0,0 +1,63 @@
+# Using Node.js with your project
+
+As you work with the sample code in this repo, you will find references to `@brightsign` JavaScript modules. These modules provide BrightSign specific functionality and are built in to the device firmware. These modules can be integrated with your code to provide rich experiences on the device.
+
+There are two main ways to run Node.js on your Brightsign device.
+
+# roHtmlWidget
+
+Create an `roHtmlWidget` object using BrightScript in your `autorun.brs` with the `nodejs_enabled` flag set to `true`. This creates a WebKit HTML renderer instance that can access the BrightSign JS modules. Set the URL of the widget to be an externally hosted page or to a file on your local filesystem. You can then set the size and position of the widget to be displayed on your device. This essentially creates a floating browser window that displays content of your choice.
+
+```
+aa=createobject("roassociativearray")
+aa.url="http://test-server/index.html"
+aa.nodejs_enabled=true
+
+r=createobject("rorectangle",0,0,1920,1080)
+h=createobject("rohtmlwidget",r,aa)
+h.show()
+```
+
+Keep in mind that any JavaScript code in your `roHtmlWidget` will only be run as long as the widget is active. You can create and display multiple instances at once.
```

Comment:
It could also be the case the graphical layer of the roHtmlWidget isn't shown because, 
1/ [show()](https://brightsign.atlassian.net/wiki/spaces/DOC/pages/370672896/roHtmlWidget#Show()-As-Boolean) is false
2/ roRectangle has x and y values which are off the screen, 
3/ or ultimately the index.html doesn't render anything and contains a "visual passthrough" or is 100% opacity.